docs/user/FAQS.md

```diff
@@ -14,6 +14,31 @@ the appropriate repository or start a discussion.
 Thank you for your willingness to contribute. Please see the
 [Getting Involved](../contrib/GettingInvolved) guide.
 
+## How do I update OpenROAD-flow scripts?
+
+Depending on how you installed OpenROAD-flow-scripts, there are different ways of updating. Regardless, here are the common first steps:
+
+```
+cd OpenROAD-flow-scripts
+git checkout master
+git pull
+```
+For WSL/docker based installation, run:
+```
+git submodule update
```

Comment:
@vvbandeira  Is required for docker based installation?

Reply:
No mode needs to run this command, it is already run by the build script. See [here](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/build_openroad.sh#L326).

Reply:
Noted.